core/imageroot/etc/systemd/system/restic-webdav.service

```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description=Restic WebDAV (rclone)
```

Comment:
There is a bit of confusion with the naming here: is it rclone or restic server?

I'd rename it Rclone WebDAV server.